### PR TITLE
Vec 341 config file support

### PR DESCRIPTION
**Pull request by jdogmcsteezy**
The installer places the asvec.yml file in /etc/aerospike like other aerospike products.

There exists 2 new flags:
- `--config-file`
- `--cluster-name` (selects which context to use in the config file)

@hev adding you to review the default config that is installed

Can't be merged until https://github.com/aerospike/tools-common-go/pull/37 us merged and go.mod is updated

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -61,30 +63,56 @@ asvec --help
...
-		if err := viper.BindPFlags(cmd.PersistentFlags()); err != nil {
+		configFile, err := config.InitConfig(rootFlags.confFile)
+		if err != nil {
 			return err
 		}
 
-		if err := viper.BindPFlags(cmd.Flags()); err != nil {
-			return err
+		if configFile != "" {
+			view.PrintfErr("Using config file: %s\n", configFile)
```
Comment (dwelch-spike) [inline cmd/root.go:74]: This is being printed as an error just to get it onto stderr, is that right? Why not log it?

**Review by dwelch-spike — COMMENTED** [review]
A small comment, otherwise LGTM

File: cmd/root.go (hunk 1 above)
Comment (jdogmcsteezy) [reply cmd/root.go:74]: At first, I wanted it to be printed out always, but now I am thinking I will just log it. Less output is better

**Review by dwelch-spike — APPROVED** [review]
LGTM

**Review by hev — APPROVED** [review]
Loos good to me.

**Pull request merged**